usaspending_api/api_contracts/contracts/agency/financial_spending/ObjectClass.md

```diff
@@ -0,0 +1,74 @@
+FORMAT: 1A
```

Comment:
Not sure I understand the directory organization - it looks like `agency/federal_obligations/FederalAccount.md` is the minor object class endpoint and so is this file. I think `FederalAccount.md` is meant to be the `/api/v2/federal_obligations/` endpoint.